src/components/FormUser/FormUser.vue

```diff
@@ -0,0 +1,169 @@
+<script lang="ts" setup>
+import { ref, reactive } from 'vue';
+import { z } from 'zod';
+import { useRouter } from "vue-router";
+import Loading from '../Loading.vue';
+
+console.log('start component user')
+
+const routerManager = useRouter();
+
+interface Message {
+  name: string;
+  lastName: string;
+  email: string;
+};
+
+interface FormData {
+  name: string,
+  lastName: string,
+  email: string,
+};
+
+const schema = z.object({
+  lastName: z.string().min(1, 'Please fill in the Last Name.').regex(/^[A-Za-z][A-Za-z\s]*$/, "Only letters are allowed"),
```

Comment:
Replace `.min` for `.required` or `.nonempty`. If both of them doesnt fix your problem, keep using `.min` and mark this comment as resolved.